recipes/aws-sam-translator/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "aws-sam-translator" %}
+{% set version = "1.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: db872c43bdfbbae9fc8c9201e6a7aeb9a661cda116a94708ab0577b46a38b962
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - boto3 ~=1.5
+    - jsonschema ~=2.6
+    - six ~=1.11
+test:
+  imports:
+    - samtranslator
+    - samtranslator.intrinsics
+    - samtranslator.model
+    - samtranslator.parser
+    - samtranslator.plugins
+    - samtranslator.policy_template_processor
+    - samtranslator.public
+    - samtranslator.sdk
+    - samtranslator.swagger
+    - samtranslator.translator
+    - samtranslator.validator
+
+
+about:
+  home: https://github.com/awslabs/serverless-application-model
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'AWS Serverless Application Model (AWS SAM) prescribes rules for expressing Serverless applications on AWS.'
+
+  # The remaining entries in this section are optional, but recommended
```

Comment:
```suggestion
```

Reply:
Oops! Thanks for catching that, will push a fix.